platform/android/src/native_map_view.hpp

```diff
@@ -88,6 +89,7 @@ class NativeMapView : public mbgl::View, private mbgl::util::noncopyable {
     JNIEnv *renderEnv = nullptr;
 
     // Ensure these are initialised last
+    std::unique_ptr<mbgl::util::RunLoop> loop;
```

Comment:
@tmpsantos Is it ok to create multiple `RunLoops` representing the main thread, if multiple map views are created? Or does this need to be a singleton somewhere else?